source/Panel.cpp

```diff
@@ -22,8 +22,8 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "GameData.h"
 #include "Point.h"
 #include "Preferences.h"
-#include "Screen.h"
 #include "text/Format.h"
+#include "Screen.h"
```

Comment:
unrelated change

Reply:
I had to, the style checker was screaming at me

Reply:
weird, let's blame @tibetiroka